src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -86,6 +87,11 @@ void GraphOptimizer::ApplyCommonGraphOptimizations(Graph &graph) {
     MergeConvertAndScaleShift(graph);
     graph.RemoveDroppedNodes();
 
+    // This optimization forces to nodes to have fp16 on directly inputs and allows them to handle precision conversion based on internal logic.
+    OV_ITT_SCOPE_NEXT(FIRST_INFERENCE, taskChain, "RemoveConvertF16");
+    RemoveConvertF16(graph);
```

Comment:
Jist realized that better to rename: FuseFCAndConvertOnWeights

Reply:
done, thanks